geowebcache/core/src/main/java/org/geowebcache/config/GeoWebCacheConfiguration.java

```diff
@@ -314,4 +314,25 @@ public Boolean getFullWMS() {
     public void setFullWMS(Boolean fullWMS) {
         this.fullWMS = fullWMS;
     }
+
+    /**
+     * If this method returns NULL CITE strict compliance mode should not be considered for WMTS
+     * service implementation.
+     *
+     * @return may return TRUE, FALSE or NULL
+     */
+    public Boolean isWmtsCiteCompliant() {
```

Comment:
Confused about the change of signature... what about keeping the field as Boolean for XStream, "boolean" for the methods, and then do a null check inside the body?

Reply:
I agree, using ``boolean`` makes the API more clean, thank you !